source/docs/running_apps.md

```diff
@@ -0,0 +1,96 @@
+
+### Running an app without starting a web server
+
+If we aren't interested in Embark starting a web server in the first place, we can easily turn it off by using the `--noserver` option:
+
+<pre><code class="shell">$ embark run --noserver</code></pre>
+
+### Switching environments
+
+Embark allows for configuring different environments to run our app against. This can be useful if we want to deploy our app to test networks or even private networks. In order to run our application in a specified environment, we first have to add a dedicated configuration to our project's `blockchain.js` file. 
+
+Depending on how you initialized your application, it may have already multiple environments configured. Here's what a sample test network environment could look like:
+
+<pre><code class="javascript">// config/blockchain.js
+
+modules.exports = {
+  ...
+  testnet: {
+    networkType: "testnet",
+    syncMode: "light",
+    account: {
+      password: "config/testnet/password"
+    }
+  },
+  ...
+}</code></pre>
+
+For more information on configuring your Embark application, head over to the [general configuration guide](configuration.html).
+
+Running an application in a different environment is just a matter of specifying the environment's name as part of the run command:
+
+<pre><code class="shell">$ embark run [environment]</code></pre>
+
+So in case we want to run our app against the test network environment described above, this could be achieved by running:
+
+<pre><code class="shell">$ embark run testnet</code></pre>
+
+### Starting a blockchain separately
+
+Sometimes we might want to have more control over the different processes involved when running our application with Embark. One of those things is to spin up a blockchain first and then have `embark run` connect to it. This enables us to stop and restart the blockchain process, without stopping Embark from doing its work.
+
+Embark comes with a `blockchain` command that does exactly that:
+
+<pre><code class="shell">$ embark blockchain</code></pre>
+
+When this is executed before Embark is run within a project, the run command will skip spinning up a blockchain node itself and connect to the existing process. Similar to the `run` command, the `blockchain` command also allows to specify an environment:
+
+<pre><code class="shell">$ embark blockchain testnet</code></pre>
+
+By default Embark blockchain will mine a minimum amount of ether and will only mine when new transactions come in.
+
+### Using the blockchain simulator
+
+Another feature of Embark is to start a **simulated** blockchain. This can be useful for testing purposes as there's no need to wait for transactions to be mined. You might have heard of [Ganache CLI](https://truffleframework.com/docs/ganache/quickstart), which is a great project that implements such a simulated blockchain.
+
+Embark integrates perfectly with this existing tool chain. To start a simulated blockchain, all we have do is to use the `simulator` command:
```

Comment:
I personally like getting the message across that we're open to collaborate and integrate with other tools. Let me know what you think about this wording.